build_library/vm_image_util.sh

```diff
@@ -421,6 +425,11 @@ _dst_path() {
 # Get the proper disk format extension.
 _disk_ext() {
     local disk_format=$(_get_vm_opt DISK_FORMAT)
+    local disk_extension=$(_get_vm_opt DISK_EXTENSION)
+    if [[ ! -z "${disk_extension}" ]]; then
```

Comment:
Quotes are not necessary in `[[` (as opposed to `[`), and `! -z` is basically `-n`:

```suggestion
    if [[ -n ${disk_extension} ]]; then
```